### ISSUE TITLE
Counting the time spent in the game

### ISSUE THREAD

**Issue by hampta** [Voxelum/x-minecraft-launcher#384]
I propose to add a calculation of playing time for each instance, each version of minecraft and everything in all versions.
![chrome_MhslRFFtqL](https://user-images.githubusercontent.com/56561445/210625841-58b63695-5011-4ffc-8e1d-36b5102525f8.png)

**Comment by CornetPanique86** [comment]
Yes, and keep the play time in a global file instead of it being in each instance. Because in Prism Launcher whenever I delete an instance the play time of it gets removed
